integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/TestDataTypePrimitive.java

```diff
@@ -395,9 +390,9 @@ public void intPath_springmvc_rt() {
 
   @Test
   public void stringPath_springmvc_rt() {
-    String expect = "serviceComb";
+    String expect = "serviceComb%2FserviceComb";
     assertEquals(expect,
-        (String) consumersSpringmvc.getSCBRestTemplate().getForObject("/stringPath/" + expect, String.class));
+        consumersSpringmvc.getSCBRestTemplate().getForObject("/stringPath/" + expect, String.class));
```

Comment:
agree to liubao
so there is some bug about this?

Reply:
Please reading https://en.wikipedia.org/wiki/Percent-encoding, I think we can add more test cases:
1. getForObject("/root/%%EF%BF%A5%promotionID"), should return an error, because this is not valide URL
2. getInf.get("%%EF%BF%A5%promotionID") should return "%%EF%BF%A5%promotionID", for RPC invocation, we can add a test case include all possible reserved characters 
3. generally, for spring mvc: any path segment should first encoded and than contact to path, and the returned value is the original string. for RPC, we get want we send.

Reply:
Spring `HierarchicalUriComponents` does duplicated encoding. As a result, the path param actually sent is "serviceComb%252FserviceComb".
To send an encoded slash, we need to build a Uri to specify path param.

Reply:
I split `stringPath_jaxrs_rt` into `stringPath_jaxrs_rt` and `stringPath_jaxrs_rt_with_encoded_slash` because I think `stringPath_jaxrs_rt_with_encoded_slash` uses new param type, it worth a separate test.